examples/websockets-realtime-voting/src/Layout.tsx

```diff
@@ -61,7 +62,9 @@ export const Layout = ({ children }: { children: ReactNode }) => {
             <Navbar.Link href="#">Contact</Navbar.Link>
           </Navbar.Collapse> */}
         </Navbar>
-        <div className="grid place-items-center mt-8">{children}</div>
+        <div className="grid place-items-center mt-8">
+          <Outlet />
```

Comment:
Breaking change: `children` -> `<Outlet />`

Reply:
Yes, children are often used as an emotional outlet for their parents, checks out.